### ISSUE TITLE
Display officially confirmed stats on statistics page as well

### ISSUE THREAD

**Issue by fossecode** [BustByte/coronastatus.no#360]
We can probably get data from one of these API's: https://github.com/soroushchehresa/awesome-coronavirus#api

Any ideas on what stats to display?

**Comment by jelbazi** [comment]
I was thinking about this too. I had the idea of maybe making a separate site, but putting it in here may be a good way to spread the information and have a stronger urge for the CTA's.

There are many stats we can show. I think these are very interesting.

For each country:
- Growth rate (moving average). Is it going up or down? For both deaths and confirmed cases.
- Cases per 1M
- Death per 1M
- Total confirmed cases
- Mortality rate
- Doubling time in days
- ICU beds per patient (if we can find this)

**Comment by jelbazi** [comment]
I have experience in time-series data manipulation. But only in python though. I can create an API that spits out the data we want. But we can also see if something exists already (from the list you shared) and call the endpoint with our current express app.

Option too is probably fastest if there's something already.

What do you think?

**Comment by fossecode** [comment]
Would be nice if you could check if it already exists an API that exposes data for all countries!

**Comment by jelbazi** [comment]
This repo is in active development and already has clear and latest data https://github.com/NovelCOVID/API

They also have an API deployed we can use directly, example for NL:
https://corona.lmao.ninja/v2/historical/netherlands

Data from the start of this year too.

**Comment by fossecode** [comment]
I think that looks good!

**Comment by jelbazi** [comment]
Yes right?! :) Javascript too, so could even implement into our own app if needed.

Shall I try to define what metrics we can derive from this data and what to show on our site?

Then discuss if everyone agrees and after that, you (or someone else) can shove it into coronastatus?

**Comment by fossecode** [comment]
Sounds like a plan!

**Comment by jelbazi** [comment]
Some basic stats I think we can glean from the above API.

Charts can be added later too, but this should be relatively simple.

![afbeelding](https://user-images.githubusercontent.com/16716173/77705074-97baf600-7023-11ea-9351-42f19beb0c4d.png)
